content/blog/2021-09-14-cats-and-dogs.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Switching the Dataset for Your Pre-trained Models
+date: 2021-09-14
+description: |
+  You can easily switch datasets for pre-trained models with DVC.
```

Comment:
It also almost repeats the title. Should we come up with some description that describes the problem?